<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemDialog</class>
 <widget class="QWidget" name="ItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar novo item ao or√ßamento</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="itemList"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="itemLabel">
     <property name="text">
      <string>Gasto mensal (R$): </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="itemValue">
     <property name="maximum">
      <double>10000000000000000000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>ItemDialog</receiver>
   <slot>addItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addItem()</slot>
 </slots>
</ui>
